src/Microsoft.AspNetCore.Razor.Language/RazorCodeDocumentExtensions.cs

```diff
@@ -3,62 +3,12 @@
 
 using System;
 using System.Collections.Generic;
-using System.Linq;
 using Microsoft.AspNetCore.Razor.Language.Intermediate;
 
 namespace Microsoft.AspNetCore.Razor.Language
 {
     public static class RazorCodeDocumentExtensions
     {
-        // Internal for testing
-        internal static readonly string IdentifierKey = "identifier";
-        internal static readonly string ImportIdentifiersKey = "imports-identifiers";
-
-        public static string GetIdentifier(this RazorCodeDocument document)
```

Comment:
Now that we have relative paths on the source document, it just makes a lot more sense to compute the identifier on demand. I wasn't a big fan of putting it on the code document, and it's possible/easy to have a mismatch between the imports documents and identifiers.